code/index.html

```diff
@@ -3,28 +3,121 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <title>Project 3 - Business Site</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&family=Work+Sans:wght@400;900&display=swap" rel="stylesheet">
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
+
+    <!-- Hero Section -->
+
+    <header class="hero-section">
+      <video class="hero-video" controls autoplay muted loop>
```

Comment:
Did you try withouth the `controls`? 😇

Reply:
Oh, I thought it was needed for the other attributes to work. Now I removed the controls and learned more about the video tag as a result, thank you :)